truss/tests/test_truss_handle.py

```diff
@@ -99,14 +99,13 @@ def test_build_docker_image(custom_model_truss_dir_with_pre_and_post):
     "base_image, expected_fail",
     [
         ("baseten/truss-server-base:3.9-v0.4.3", False),
-        ("baseten/truss-training-base:3.9-v0.4.3", False),
```

Comment:
you could add this back in I think

Reply:
this builds the serving image instead of the training image – I made a separate test for building the training image